frontend/src/ui/Blocks/Series.tsx

```diff
@@ -193,43 +210,364 @@ type SeriesBlockContainerProps = {
     children: ReactNode;
 };
 
-const SeriesBlockContainer: React.FC<SeriesBlockContainerProps> = ({ title, children }) => (
-    <div css={{
-        marginTop: 24,
-        padding: 12,
-        ...title && { paddingTop: 0 },
-        backgroundColor: "var(--grey95)",
-        borderRadius: 10,
-    }}>
-        {title && <div css={{
-            display: "inline-block",
-            padding: "8px 12px",
-            marginTop: 12,
-        }}>
-            <WithTooltip tooltip={title}>
-                <h2 css={{
-                    color: "var(--grey40)",
-                    fontSize: 19,
-                    lineHeight: 1.3,
-                    ...ellipsisOverflowCss(2),
-                }}>{title}</h2>
-            </WithTooltip>
-        </div>}
-        {children}
-    </div>
+type View = "slider" | "gallery" | "list";
+
+const ViewContext = createContext({
+    viewState: "gallery",
+    setViewState: (_newState: View) => {},
+});
+
+const SeriesBlockContainer: React.FC<SeriesBlockContainerProps> = ({ title, children }) => {
+    const [viewState, setViewState] = useState<View>("gallery");
+
+    return (
+        <ViewContext.Provider value={{ viewState, setViewState }}>
+            <div css={{
+                marginTop: 24,
+                padding: 12,
+                ...title && { paddingTop: 0 },
+                backgroundColor: "var(--grey95)",
+                borderRadius: 10,
+            }}>
+                <div css={{
+                    display: "flex",
+                    justifyContent: "space-between",
+                    paddingTop: 14,
```

Comment:
I could give the container of the buttons a `margin-right: 5` or `padding-right: 5` to fix this (the correct value would actually be `4.85` but I think that's basically indistinguishable from `5`).
Or better maybe to just do `padding: 5` and your suggestion of `align-self: flex-start`. This would keep the buttons aligned with a single line title and also make them stick to the top with multi line titles, which I think looks better than having them vertically centered.

Reply:
Yeah I think I like the latter. Works well for single line titles and sticks to the top for multi line ones.